meveo-admin/ejbs/src/main/java/org/meveo/export/EntityExportImportService.java

```diff
@@ -2189,6 +2193,7 @@ private void refreshCaches() {
         notificationCacheContainerProvider.refreshCache(null);
         customFieldsCacheContainerProvider.refreshCache(null);
         jobCacheContainerProvider.refreshCache(null);
+        liquichainCacheContainerProvider.refreshCache(null);
```

Comment:
Maybe it's better not to mention liquichain here and rename this to userMessageCacheContainerProvider

Reply:
renamed